### DISCUSSION TITLE
M2A fields don't display content in version 9.2.2

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#10560] [Archive]
I found that M2A fields don't display content and are always empty in the newest 9.2.2 version.

After I downgraded to 9.1.2, the problem disappeared. 

I think the bug is only about display, because I can still add and remove records for these fields, though the changes are not reflected in the GUI.

Please take a look. Thanks!

**Comment by rijkvanzanten** [comment]
Heya! 

This sounds like a (potential) bug. Mind [opening a new issue](https://github.com/directus/directus/issues/new)? Thanks!

**Reply by powerseed** [reply]
Done. 
https://github.com/directus/directus/issues/10601
Thanks!
